scripts/clone/services.sh

```diff
@@ -7,14 +7,15 @@ source ./scripts/clone/utils.sh
 cd omniport/services/
 
 # Clone services
-clonerepo   "Apps"          "omniport-frontend-apps"            "apps"
-clonerepo   "Auth"          "omniport-frontend-auth"            "auth"
-clonerepo   "Developer"     "omniport-frontend-developer"       "developer"
-clonerepo   "Feed"          "omniport-frontend-feed"            "feed"
-clonerepo   "File Manager"  "omniport-frontend-file-manager"    "file_manager"
-clonerepo   "Groups"        "omniport-frontend-groups"          "groups"
-clonerepo   "Helpcentre"    "omniport-frontend-helpcentre"      "helpcentre"
-clonerepo   "Links"         "omniport-frontend-links"           "links"
-clonerepo   "OAuth"         "omniport-frontend-oauth"           "oauth"
-clonerepo   "Settings"      "omniport-frontend-settings"        "settings"
-clonerepo   "Terms of use"  "omniport-frontend-terms-of-use"    "terms_of_use"
+clonerepo   "Apps"            "omniport-frontend-apps"            "apps"
```

Comment:
@kmrinal19 you have mistakingly indented all the app names. Make sure you correct it.